python/heterocl/tvm/build_module.py

```diff
@@ -462,7 +467,7 @@ def build_fpga_kernel(sch, args, target, name="default_function", schedule_name=
                 assert m in ["csyn", "csim", "cosim", "impl", "custom"], \
                     "not supported mode " + m
         else:
-            assert mode in ["csyn", "csim", "cosim", "impl", "custom",
+            assert mode in ["csyn", "csim", "cosim", "impl", "custom", "power",

```

Comment:
What is `power`? Are there any more specific names here?

Reply:
`power` stands for power analysis in Stratus HLS. `power` for power analysis is consistent with Stratus's terms, e.g. one runs `make power_P_BASIC` for power analysis results. I think it's fine.